frontend/src/components/Client/TextSection/TextSection.js

```diff
@@ -0,0 +1,10 @@
+const TextSection = ({title, content}) => {
+  return (
+    <div className="h-14">
+      <span className="text-xl pt-3">{title}</span>
+      <p className="text-sm line-clamp-4 text-stone-600 md:line-clamp-6">{content}</p>
```

Comment:
My bad, I forgot to mention that I have updated the default text color to text-gray-700 in the index.css file so there is no need to add individual colors to any default text in any component unless we want to customize specifically.